vividus-plugin-web-app-to-rest-api/src/main/java/org/vividus/steps/integration/ResourceCheckSteps.java

```diff
@@ -216,6 +218,14 @@ private String getCssSelector(Element element)
     private URI resolveUri(String uri) throws URISyntaxException
     {
         URI uriToCheck = new URI(uri);
+
+        if (uri.startsWith("//") && uriToCheck.getHost() != null)
+        {
+            String scheme = Optional.ofNullable(webApplicationConfiguration.getMainApplicationPageUrlUnsafely())
+                    .map(URI::getScheme).orElse(HTTPS);
```

Comment:
why is default value `https`?

Reply:
what would you suggest if not https? this is the most widely used